libs/natparse/src/main/java/org/amshove/natparse/lexing/Lexer.java

```diff
@@ -904,7 +973,7 @@ private boolean consumeComment()
 				|| lookahead == '\r'
 				|| lookahead == '/'
 				|| lookahead == SourceTextScanner.END_CHARACTER);
-		var isInlineComment = scanner.peek() == '/' && lookahead == '*';
```

Comment:
The variable `isSingleAsteriskComment` above is now unused and can be removed.

Reply:
I am using it in  
989 if (isSingleAsteriskComment() || isInlineComment)

Reply:
In that case it is a method invocation, the variable is still unused and can be removed

Reply:
Oh, I see your point